frontend/js/src/explore/artist-similarity/artist-search/artistLookup.ts

```diff
@@ -0,0 +1,34 @@
+type ArtistType = {
+  name: string;
+  id: string;
+  type?: string;
+  country?: string;
+};
+
+type ArtistLookupResponseType = {
+  created: string;
+  count: number;
+  offset: number;
+  artists: Array<ArtistType>;
+};
+
+const artistLookup = async (searchQuery: string) => {
+  let resultsArray: Array<ArtistType> = [];
+  const LOOKUP_URL = `https://musicbrainz.org/ws/2/artist/?query=artist:${searchQuery}&fmt=json`;
+  let data: ArtistLookupResponseType;
+  // Fetches the artists from the MusicBrainz API
+  try {
+    const response = await fetch(LOOKUP_URL);
+    data = await response.json();
+    resultsArray = data.artists;
+  } catch (error) {
+    alert(error);
```

Comment:
remove alert. You can throw an error here and capture it in your components to show an error toast.

Reply:
Fixed